lms/static/styles/components/_FileList.scss

```diff
@@ -58,3 +60,11 @@ $file-list-icon-size: 24px;
 .FileList__date-col {
   padding-right: 10px;
 }
+
+.FileList__no-files {
+  border: 1px solid var.$grey-6;
+  height: 40px;
+  vertical-align: middle;
+  padding: 12px;
+  background-color: #eee;
```

Comment:
Perhaps use `var.$grey-2` here (#ececec) for consistency with the disabled buttons in the form. Using color variables for foreground and background colors also makes it easier to check that contrast is high enough everywhere.

Reply:
I was making it consistent with the look and feel of the zebra-stripe rows in the normal table (https://github.com/hypothesis/lms/blob/c0fa85c9a79186b3efda91e82a15ce5fb763350b/lms/static/styles/lms.css#L193).

Sure, I will make it `var.$grey-2`